<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>147</green>
        <blue>30</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>147</green>
        <blue>30</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="216">
        <red>248</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="216">
        <red>248</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>147</green>
        <blue>30</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>147</green>
        <blue>30</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="216">
        <red>248</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="216">
        <red>248</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>147</green>
        <blue>30</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Hanson</family>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Free Throw</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>540</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hanson</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>THE PRACTICE SCORE TRACKER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>440</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hanson</family>
     <pointsize>30</pointsize>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,147,30);
color: white;
</string>
   </property>
   <property name="text">
    <string>START GAME</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>540</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hanson</family>
     <pointsize>49</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>FREE THROW</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>440</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hanson</family>
     <pointsize>30</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,147,30);
color: white;
</string>
   </property>
   <property name="text">
    <string>EDIT PLAYERS</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>700</y>
     <width>440</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hanson</family>
     <pointsize>30</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,147,30);
color: white;
</string>
   </property>
   <property name="text">
    <string>VIEW STATS</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
